internal/handler/sandbox.go

```
@@ -225,19 +233,57 @@ func sandboxBook(ctx context.Context, rc *runner.RunContext, client *clients.San
-		}, nil
-	case 202, 507:
-		// Queued or no capacity.
-		return &SandboxResult{Status: "queued", Placement: result}, nil
+	case http.StatusOK, http.StatusAccepted:
+		switch placement.Status {
+		case "success", "complete":
+			// Re-convert SandboxPlacement to map[string]interface{} for extract helpers
+			// which expect maps to process generic resource fields.
+			var pMap map[string]interface{}
+			pBytes, _ := json.Marshal(placement)
+			json.Unmarshal(pBytes, &pMap)
```

golangci-lint (2.13.2)

[error] Error return value of `json.Unmarshal` is not checked

(errcheck)